build-scratch

```diff
@@ -1,4 +1,8 @@
 #!/bin/sh
-SHA=`git rev-parse HEAD`
-docker build -t $SHA . &&
-    docker run --rm --entrypoint tar $SHA -c /usr/bin/ct Dockerfile.build-scratch | docker build -f Dockerfile.build-scratch --tag coreos/ct -
+
+set -e
+VERSION=$(./git-version)
+docker build -t $VERSION .
+docker run --rm --entrypoint tar $VERSION -c /usr/bin/ct Dockerfile.build-scratch \
+    | docker build -f Dockerfile.build-scratch --tag quay.io/coreos/ct:$VERSION -
+docker tag quay.io/coreos/ct:$VERSION quay.io/coreos/ct:latest-dev
```

Comment:
When I say "hash", I'm referring to the sha1. I see now though, that by pushing the hash/tag, we will automatically get tagged images instead of having to do something different for the release process. This seems fine.